src/Auth0.Core/ApiError.cs

```diff
@@ -27,10 +28,23 @@ public class ApiError
         [JsonProperty("message")]
         public string Message { get; set; }
 
-        /// <summary>
-        /// The HTTP Status code.
-        /// </summary>
-        [JsonProperty("statusCode")]
-        public int StatusCode { get; set; }
+        public static ApiError ParseOrDefault(string responseContent)
+        {
+            if (String.IsNullOrEmpty(responseContent))
+                return default;
+
+            try
+            {
+                return JsonConvert.DeserializeObject<ApiError>(responseContent);
```

Comment:
Is this an API error from our server? Is it expecting a specific format for this JSON?

Reply:
There is an ApiError deserializer that was already part of the code base to create ApiErrors from our responses.